server/api/crud/model_monitoring/deployment.py

```diff
@@ -395,30 +281,21 @@ def _initial_model_monitoring_stream_processing_function(
 
         return function
 
-    def _get_model_monitoring_batch_function(
-        self, image: str, function_name: str = "model-monitoring-batch"
-    ):
+    def _get_model_monitoring_batch_function(self, image: str, function_name: str):
         """
         Initialize model monitoring batch function.
 
-        :param image:                       Base docker image to use for building the function container
-        :param function_name:               model-monitoring-batch or model-monitoring-controller
-                                            indicates witch one to create.
-        :return:                            A function object from a mlrun runtime class
+        :param image:         Base docker image to use for building the function container
+        :param function_name: "model-monitoring-controller".
```

Comment:
I think you can delete this param and to use the constant inside this function - not mandatory.

Reply:
Done in fc85c1c